### ISSUE TITLE
Support for timezone names

### ISSUE THREAD

**Issue by robertdumitrescu** [taylorhakes/fecha#84]
Hey, 

Can you please add support for timezone names such as ```GMT```,  ```UTC```, ```EST```, ```BST``` and so on? 

I really love your library and the fact that is so small and performant and that would help me greatly to parse logs from various servers around the world. 

Thank you!

**Comment by taylorhakes** [comment]
I agree this would be useful, but this would significantly increase the size of this library to hold the entire mapping. I personally feel the tradeoff is not worth it, but happy to be convinced.

**Comment by taylorhakes** [comment]
Closing for now. Thanks for the suggestion
